backendApi/src/main/java/com/virtuslab/gitmachete/backend/api/SyncToRemoteStatus.java

```diff
@@ -5,7 +5,7 @@
 @Data(staticConstructor = "of")
 public class SyncToRemoteStatus {
   public enum Relation {
-    Untracked, Ahead, Behind, DivergedAndNewerThanRemote, DivergedAndOlderThanRemote, InSync
+    Untracked, AheadOfRemote, BehindRemote, DivergedAndNewerThanRemote, DivergedAndOlderThanRemote, InSync
```

Comment:
While we are there - to be consistent
```suggestion
    Untracked, AheadOfRemote, BehindRemote, DivergedAndNewerThanRemote, DivergedAndOlderThanRemote, InSyncWithRemote
```
or maybe better

```suggestion
    Untracked, Ahead, Behind, DivergedAndNewer, DivergedAndOlder, InSync
```

Reply:
I'd favor the former (with `...WithRemote`); in the latter, `Ahead`, `Behind` etc. will get confusing quickly (what is ahead of what)

Reply:
do trzech razy sztuka :rofl: